tasks/vulcanize.js

```diff
@@ -18,17 +20,14 @@ module.exports = function(grunt) {
     var done = this.async();
     // Merge task-specific and/or target-specific options with these defaults.
     var options = this.options({
-      abspath: null,
-      config: null,
-      csp: false,
-      inline: false,
-      strip: false,
-      'strip-excludes': true,
-      excludes: {
-        imports: [],
-        scripts: [],
-        styles: []
-      }
+      abspath: '',
+      excludes: [
+      ],
+      stripExcludes: false,
```

Comment:
This should be an array. `stripExcludes` and `excludes` are the same thing when `implicitStrip` is `true`.